src/discordRPC/runRPC.bat

```diff
@@ -2,7 +2,7 @@ set "documentsPath=C:\Users\%USERNAME%\Documents\Paradox Interactive\Hearts of I
 @REM This is file for executing the game and the presence, delete if you have uninstalled and change launcher-settings.json
 
 echo "starting hoi4RPC..."
-start "hoi4.exe" "./hoi4.exe"
+start "hoi4.exe" "%documentsPath%\hoi4.exe"
```

Comment:
This is the executable file of the game, and it is not in documents path. Revert it